docs/introduction/what_is_chrysalis.md

```diff
@@ -42,7 +42,7 @@ A new [Uniform random tip selection](https://github.com/iotaledger/protocol-rfcs
 ![](./assets/what_is_chrysalis/07.png)
 [A switch to an internal binary representation of the trinary transaction](https://github.com/luca-moser/protocol-rfcs/blob/signed-tx-payload/text/0000-transaction-payload/0000-transaction-payload.md). This allows us to work on binary data for validation, IO, and other processing without the current reliance on binary-ternary conversions as in the pre-Chrysalis software node software. The switch to binary transactions further reduces the transaction size, saving network bandwidth and processing time.
 
-## New node API and client libraries
-With Chrysalis, we wanted to offer a more standard API on both the node and client library level. Node implementations provide a completely redesigned [RESTful](https://editor.swagger.io/?url=https://raw.githubusercontent.com/rufsam/protocol-rfcs/master/text/0026-rest-api/rest-api.yaml) and [eventful](https://playground.asyncapi.io/?load=https://raw.githubusercontent.com/luca-moser/protocol-rfcs/rfc/node-event-api/text/0033-node-event-api/0033-node-event-api.yml) API implementations.
+## New Node API and Client Libraries
+With Chrysalis, we wanted to offer a more standard API on both the node and client library level. Node implementations provided a completely redesigned [RESTful](https://editor.swagger.io/?url=https://raw.githubusercontent.com/rufsam/protocol-rfcs/master/text/0026-rest-api/rest-api.yaml) and [eventful](https://playground.asyncapi.io/?load=https://raw.githubusercontent.com/luca-moser/protocol-rfcs/rfc/node-event-api/text/0033-node-event-api/0033-node-event-api.yml) API implementations.
```

Comment:
I think the idea is that the Node now provides a completely redesigned RESTful and eventful API implementations, so I'd revert this change in line 46.   If it's past tense it implies this doesn't happen anymore